packages/relay/src/formatters.ts

```diff
@@ -79,6 +79,21 @@ const formatTransactionId = (transactionId: string): string | null => {
   return `${payer}-${timestamp}`;
 };
 
+/**
+ * Formats a transaction object by moving input to data, if data is absent.
```

Comment:
Would you consider adding an explanation for why such a transformation is necessary here? I mean the PR description explains it very well, and I think incorporating that explanation here in the codebase would make it a good reference for this method, and help developers to understand why it's important without looking into the PR to find the description.